### PR TITLE
Add demoer demos

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

**Comment by keirog** [comment]
Demo convention comments welcome!

**Comment by ironsidevsquincy** [comment]
DEMO-CON 2015!

**Comment by keirog** [comment]
YES

File: demos/data.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+			"url": "/stream/topicsId/Mzg0ZGEwYWMtNTc2Yi00M2QwLTljZjMtMWFmNTEyOWJlMTA3-VG9waWNz"
+		},
+		{
+			"demoHeading": "Topic bar with annotation",
+			"name": "Angela Merkel",
+			"url": "/stream/peopleId/TnN0ZWluX1BOX0FGVE1fUE5fMTMwMjM=-UE4=",
+			"role": {
+				"title": "Chancellor, Government of Germany"
+			}
+		}
```
Comment (wheresrhys) [inline demos/data.json]: Would be nice to have one fro a company too to demonstrate the ticker symbol & js. Beoing co from the foot of the page here is a good example of the data http://next.ft.com/db1cab82-e8ec-11e4-87fe-00144feab7de

Comment (keirog) [inline demos/data.json]: Ah, I was avoiding that one due to favouritism-related concerns

Comment (wheresrhys) [inline demos/data.json]:
```
<div class="n-topic n-topic--bar n-topic--follow n-topic--ticker" data-taxonomy="organisations" data-concept-id="organisations:&quot;Pearson%20PLC&quot;" data-ticker-symbol="uk:PSON">
				<h2 class="n-topic__name">
						<a class="n-topic__link" href="/stream/organisationsId/T05fbnlzZS1leHRyYV9mZXYtMjAwOF8xMzAx-T04%3D" data-trackable="topic-link">
							Pearson PLC
						</a>
				</h2><p class="n-topic__ticker n-topic__annotation">
							<a href="https://markets.ft.com/data/equities/tearsheet/summary?s=PSON:LSE" data-trackable="markets-link">PSON:LSE</a>
							<span class="market-data__price">1294</span>
							<span class="market-data__change">+0.54%</span>
						</p>
					<form class="myft-ui myft-ui--follow" action="/__user-prefs/activities/followed/Topic:organisations%3A%22Pearson%2520PLC%22?redirect=/mypage/topics?conceptId=organisations:&quot;Pearson%20PLC&quot;%26verb=followed" method="POST">
					<button type="submit" class="myft-ui__button" data-concept-id="organisations:&quot;Pearson%20PLC&quot;" data-trackable="follow" data-myft-button="follow" data-alternate-label="Following this topic" data-alternate-text="Following" aria-label="Follow this topic" title="Follow this topic">Follow</button>
					</form>
			
				
				
			
			</div>
```

Comment (keirog) [inline demos/data.json]: Ah, thanks and sorry… I only checked the FT, which didn't have it.

**Pull request merged**